src/main/webapp/app/exercises/shared/participation/participation.component.html

```diff
@@ -221,7 +221,7 @@ <h2>
                     <ng-template ngx-datatable-cell-template let-value="value">
                         <div class="w-100 text-right">
                             <div class="btn-group">
-                                <ng-container *ngIf="exercise?.type === ExerciseType.PROGRAMMING && exercise.isAtLeastInstructor">
+                                <ng-container *ngIf="exercise?.type === ExerciseType.PROGRAMMING && (exercise.isAtLeastInstructor || exercise.exerciseGroup)">
```

Comment:
You can get the course through exercise -> exerciseGroup -> exam -> course

Reply:
I guess the attribute exercise.isAtLeastInstructor is set when downloading the exercise in the client side service. I think we should just compute the value while downloading the exercises, then everything should be fine